spec/spec_helper.rb

```diff
@@ -1,8 +1,43 @@
+ENV['RACK_ENV'] = 'test'
+
 require 'coveralls'
 require 'simplecov'
+require 'capybara'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.